mkosi.md

```diff
@@ -1152,6 +1152,11 @@ a boolean argument: either "1", "yes", or "true" to enable, or "0",
   authorized keys in the built image. The `ssh` invocation done by `mkosi ssh`
   will inherit `$SSH_AUTH_SOCK` for authentication purposes.
 
+`SshPort=`, `--ssh-port=`
+
+: If specified, when `NetworkVeth=` is disabled, use this port to connect via
+  `mkosi ssh` on localhost.
```

Comment:
It's a bit unclear if this is the originating port or the destination port. Also, is this actually conditional on NetworkVeth? Should this just say "In the image, sshd will be configured to listen on this port. 'mkosi ssh' will connect to this port." ?

Reply:
ok, updated